### ISSUE TITLE
Website accessibility testing

### ISSUE THREAD

**Issue by Aurelius333** [WalnutProgramming/Directions#92]
I've tried to make our site accessible, but someone should probably see whether our site is usable with a screen reader. 

We might need to look at stuff like this: <https://marcus.io/blog/improved-accessible-routing-vuejs>.

We should add skip links.

(This is about making the website accessible, not about making the directions accessible - see #91)

**Comment by presssssure** [comment]
While we should definitely make the website screen reader accessible, I don't think it should be top priority. The directions aren't really built with the visually impaired in mind. "Turn left after room 1319" wouldn't be that useful to someone who can't really see room 1319.

Of course they could still hear about the website and want to check it out, but probably not actually use it.

**Comment by Aurelius333** [comment]
I agree @presssssure

**Comment by Aurelius333** [comment]
Low priority (see @presssssure's comment). Closing for now.
